### DISCUSSION TITLE
Help Needed: Serverpod Authentication Issue with Google Sign-In

### DISCUSSION THREAD

**Discussion by sachu-seqato** [serverpod/serverpod#2524] [Q&A]
Hi everyone,

I am new to Serverpod and currently working on developing backend implementation using it. Now, I am encountering an authentication issue and would appreciate any help.

The issue is:

`I/flutter (15146): serverpod_auth_google: PlatformException(sign_in_failed, com.google.android.gms.common.api.ApiException: 10: , null, null)`

I have already tried this on two different machines but am facing the same issue. I followed the Serverpod documentation as well.

Has anyone faced a similar issue or know how to resolve it? Your assistance would be greatly appreciated.

Thanks in advance!

**Comment by jaxfire** [comment]
You may possibly need to regen your google_client_secret.json and replace your local version.

https://stackoverflow.com/a/78637154

**Comment by bphd** [comment]
E/Finsky  (): [173] uks.a(849): Encountered exception whilst fetching from GMS, deferring to Finsky: com.google.android.gms.common.api.ApiException: 17: API: IntegrityApiDisplayListener.API is not available on this device. Connection failed with: ConnectionResult{statusCode=API_DISABLED, resolution=null, message=null}
